Dockerfile

```diff
@@ -1,11 +1,25 @@
-FROM golang:1.18 as workspace
+# dummy value for linux builds
+ARG BASE_IMAGE=scratch
+
+FROM --platform=${BUILDPLATFORM} golang:1.18 AS workspace
```

Comment:
```suggestion
FROM --platform=${BUILDPLATFORM} golang:1.19 AS workspace
```

Reply:
We are still in Go 1.18, so I'd keep it as is, and use a follow-up PR to bump the Go version 🙏